index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Sonar for Bitbucket API Specification</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+      margin: 0;
+      padding-top: 40px;
+    }
+
+    nav {
+      position: fixed;
+      top: 0;
+      width: 100%;
+      z-index: 100;
+    }
+    #links_container {
+      margin: 0;
+      padding: 0;
+      background-color: #7E4A95;
+    }
+    /*prevent distracting download spec button*/
+    .download-openapi {
+      display: none !important;
+    }
+    #links_container li {
+      display: inline-block;
+      padding: 10px;
+      color: white;
+      cursor: pointer;
+      font-family: Arial, Helvetica, sans-serif;
+    }
+  </style>
+</head>
+<body>
+
+<!-- Top navigation placeholder -->
+<nav>
+  <ul id="links_container">
+  </ul>
+</nav>
+
+<redoc scroll-y-offset="body > nav"></redoc>
+
+<script src="https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.js"> </script>
```

Comment:
Not sure if we want to stick with `latest`, as it can cause incompatibilities without that we would notice?

```suggestion
<script src="https://cdn.jsdelivr.net/npm/redoc@v2.0.0-rc.66/bundles/redoc.standalone.js"> </script>
```